medusalib/src/test/java/com/trendyol/medusalib/navigator/FragmentStackStateBuilder.kt

```diff
@@ -0,0 +1,42 @@
+package com.trendyol.medusalib.navigator
+
+import com.trendyol.medusalib.navigator.data.StackItem
+import java.util.*
```

Comment:
Please use a certain class instead of wildcard

Reply:
done